### PR TITLE
WIP feat(DTFS2-5659): extend component tests tfm ui

### PR DESCRIPTION
**Pull request by vamshidharukef**
Introduction ✏️
Improve coverage for component tests for TFM UI

Resolution ✔️
Added more component tests for TFM UI

### CODE REVIEW

File: trade-finance-manager-ui/component-tests/case/activity/_macros/activity.component-test.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const pageRenderer = require('../../../component-tests/pageRenderer');
+const pageRenderer = require('../../../pageRenderer');
 
 const page = '../templates/case/activity/activity.njk';
 const render = pageRenderer(page);
```
Comment (oscar-richardson-softwire) [inline trade-finance-manager-ui/component-tests/case/activity/_macros/activity.component-test.js:4]: Has this file definitely been moved to the right location to mirror the structure of the templates? The template this file is testing is in `trade-finance-manager-ui\templates\case\activity\activity.njk` (i.e, not inside `\_macros`), but this test file is in `trade-finance-manager-ui\component-tests\case\activity\_macros\activity.component-test.js`.

File: trade-finance-manager-ui/component-tests/case/tasks/_macros/tasks-table.component-test.js
Comment (oscar-richardson-softwire) [inline trade-finance-manager-ui/component-tests/case/tasks/_macros/tasks-table.component-test.js:1]: Can we simplify the import in this file from `'../../../../component-tests/componentRenderer'` to `'../../../componentRenderer'` please?

File: trade-finance-manager-ui/component-tests/case/underwriting/pricing-and-risk1/_macros/section-facilities/facility-pricing-risk-table.component-test.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const componentRenderer = require('../../../../../../component-tests/componentRenderer');
+const componentRenderer = require('../../../../../componentRenderer');
 
 const component = '../templates/case/underwriting/pricing-and-risk/_macros/section-facilities/facility-pricing-risk-table.njk';
```
Comment (oscar-richardson-softwire) [inline trade-finance-manager-ui/component-tests/case/underwriting/pricing-and-risk1/_macros/section-facilities/facility-pricing-risk-table.component-test.js:4]: Why has the folder been named 'pricing-and-risk1'?

**Review by MarRobSoftwire — COMMENTED** [review]
It looks like the following still aren't covered by tests
* `case/`
    * `ammendments`
    * `facility/_macros`
    * `parties/edit`
    * `parties/summary`
* `facilities`
* `feedback`
* `user`  
Will this be covered in a future PR?

File: trade-finance-manager-ui/component-tests/case/underwriting/pricing-and-risk1/_macros/section-exporter/section-exporter.component-test.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-const componentRenderer = require('../../../../../../component-tests/componentRenderer');
+const componentRenderer = require('../../../../../componentRenderer');
 
 const component = '../templates/case/underwriting/pricing-and-risk/_macros/section-exporter/section-exporter.njk';
```
Comment (oscar-richardson-softwire) [inline trade-finance-manager-ui/component-tests/case/underwriting/pricing-and-risk1/_macros/section-exporter/section-exporter.component-test.js:4]: Why has the folder been named `'pricing-and-risk1'`?

Comment (vamshidharukef) [reply trade-finance-manager-ui/component-tests/case/underwriting/pricing-and-risk1/_macros/section-exporter/section-exporter.component-test.js]: Renamed the folder to `pricing-and-risk`

File: trade-finance-manager-ui/component-tests/case/underwriting/pricing-and-risk1/_macros/section-facilities/facility-pricing-risk-table.component-test.js (hunk 2 above)
Comment (vamshidharukef) [reply trade-finance-manager-ui/component-tests/case/underwriting/pricing-and-risk1/_macros/section-facilities/facility-pricing-risk-table.component-test.js]: Renamed the folder to `pricing-and-risk`

File: trade-finance-manager-ui/component-tests/case/underwriting/pricing-and-risk1/_macros/section-exporter/exporter-table.component-test.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-const componentRenderer = require('../../../../../../component-tests/componentRenderer');
+const componentRenderer = require('../../../../../componentRenderer');
 
 const component = '../templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk';
```
[the remark this answers is not in the document]
Comment (vamshidharukef) [reply trade-finance-manager-ui/component-tests/case/underwriting/pricing-and-risk1/_macros/section-exporter/exporter-table.component-test.js]: Renamed folder to `pricing-and-risk`

File: trade-finance-manager-ui/component-tests/case/activity/_macros/activity.component-test.js (hunk 1 above)
Comment (vamshidharukef) [reply trade-finance-manager-ui/component-tests/case/activity/_macros/activity.component-test.js]: Moved the test under` trade-finance-manager-ui\component-tests\case\activity\activity.component-test.js`

**Comment by vamshidharukef** [comment]
With the number of changed files on this PR there might be a possibility the functionality might break if merged to main so deleting and creating a new PR https://github.com/UK-Export-Finance/dtfs2/pull/2460 with the number of files changed  using `git cherry-pick.`

**Pull request closed without merging**